### PR TITLE
RK-11413 - Remove perforce from desktop app

### PR DESCRIPTION
**Pull request by DavidRBiderman**
Removed all references to perforce from explorook desktop app

### CODE REVIEW

File: src/index-worker.ts [hunk 1]
```diff
@@ -6,7 +6,6 @@ import { Repository } from "./common/repository";
 import { initExceptionManager, notify } from "./exceptionManager";
 import {cloneRemoteOriginWithCommit, GitConnectionOptions} from "./git";
 import {getLogger, setLogLevel} from "./logger";
```
Comment (ron-rookout) [inline src/index-worker.ts:8]: Unused import

Comment (DavidRBiderman) [reply src/index-worker.ts]: Removed unused imports

File: src/api.ts [hunk 2]
```diff
@@ -378,53 +284,16 @@ export const resolvers = {
...
-        logger.error("Could not get Perforce manager instace");
-        return null;
-      }
-
-      logger.debug("Getting changelist for file", args);
-      return perforceManager.getChangelistForFile(fileFullpath);
-    },
     getCommitIdForFile: async (parent: any, args: {provider: any, remoteOrigin: string, repo: Repository, path: string}): Promise<string> => {
       logger.debug("Getting commit ID for file", args);
       const {provider, repo, path, remoteOrigin} = args;
```
Comment (ron-rookout) [inline src/api.ts:289]: Unused variable

**Review by ron-rookout — APPROVED** [review]

**Pull request merged**